test/e2e-kind.sh

```diff
@@ -22,13 +23,17 @@ run_ct_container() {
         teamcity_volume=(-v /teamcity/system/git:/teamcity/system/git)
     fi
     docker run --rm --interactive --detach --network host --name ct \
-        --volume "$(pwd)/test/ct-e2e.yaml:/etc/ct/ct.yaml" \
-        --volume "$(pwd):/workdir" \
         "${teamcity_volume[@]}" \
-        --workdir /workdir \
+        --workdir /charts \
         "quay.io/helmpack/chart-testing:${CT_VERSION}" \
         cat
     echo
+
+    docker cp "$(pwd)/test/ct-e2e.yaml" "ct:/etc/ct/ct.yaml"
+    docker cp "$(pwd)" "ct:/charts"
```

Comment:
```suggestion
    docker cp "$(pwd)/" "ct:/charts"
```
This should fix it.

Reply:
let's give that a shot thanks max :+1:

Reply:
Thanks for trying @makkes :slightly_smiling_face:  Turns out I needed to copy everything _in_ the current working directory, i.e., `$pwd)/.`

Can you approve?